src/cart/crt_init.c

```diff
@@ -215,26 +219,6 @@ prov_data_init(struct crt_prov_gdata *prov_data, crt_provider_t provider,
 
 	D_DEBUG(DB_ALL, "Max number of contexts set to %d\n", max_num_ctx);
 
-	/* Assume for now this option is only available for a primary provider */
```

Comment:
Note: This section is going away in next patch anyway, as we no longer support SEP mode anywhere. Decided to nuke it here as well, instead of trying to rework the code that will get nuked